ui/src/Shared/services/login-service.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+import { User } from '../models/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginServiceService {
+  currUser: User = { email: '', password: '' };
+  setLogout = new Subject<boolean>();
+  dataArray: User[] = [];
```

Comment:
You can name this `userDetails` or `userList`, or just `users` to make the intent clear

Reply:
Okay. Resolved the issue as suggested.